src/components/tokens/_tokens.scss

```diff
@@ -1,6 +1,6 @@
 /**
  * Do not edit directly
- * Generated on Tue, 03 May 2022 22:07:09 GMT
+ * Generated on Wed, 04 May 2022 03:10:35 GMT
```

Comment:
I had to fix a merge conflict here. On the YaleSites project, we're not actually committing the built tokens (this file) I have the GitHub workflow:
- build the file and force commit it
- cut a release
- then remove it from git again

That way we avoid merge conflicts. We should discuss whether we want to do that here too.